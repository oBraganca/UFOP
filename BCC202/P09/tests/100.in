100
Kauê 784637 1.6
Larissa 122344 3.3
Theo 202437 9.5
Nicolas 607739 5.0
Lucca 747818 6.7
Bruno 39210 6.3
LuizMiguel 975866 0.7
Erick 755993 4.3
Gael 601185 1.5
Agatha 942503 3.3
JoaoMiguel 631594 7.0
VictorHugo 3713 1.3
Thomas 416182 5.2
Ian 278622 1.9
JoaoLucas 631990 5.0
Lais 778936 9.6
Joana 812356 1.9
Alice 233725 8.7
Caua 979635 7.6
Francisco 319887 9.8
Lucas 827345 8.8
Fernanda 17160 3.3
AnaLiz 310209 8.1
Emanuel 382454 1.4
LuizGustavo 358286 5.3
Enzo 802024 3.6
LucasGabriel 38609 1.3
Elisa 550011 2.3
Maitê 782422 9.5
Marina 44918 3.0
Ruan 778027 8.0
Arthur 775731 10.0
AnaSophia 15646 9.6
Bernardo 168265 4.8
Benicio 617490 8.0
Bianca 106958 1.2
Pietra 31845 4.9
MariaFernanda 761050 2.1
Victor 752759 7.0
PedroMiguel 317302 2.8
Samuel 924683 7.9
JoaoGabriel 783559 2.4
Livia 410218 2.5
Leonardo 289192 9.3
Gabriela 909372 2.1
Thiago 862749 8.5
AnaLaura 73060 4.0
JoaoGuilherme 403042 0.6
Luna 605714 8.1
MariaEduarda 423549 5.9
Allana 447784 9.3
Isadora 537797 8.8
Ryan 752634 1.8
Joao 378948 4.1
Joaquim 187555 9.3
Henry 613407 7.5
Isis 25792 5.6
Amanda 153881 4.6
Mariah 376893 4.8
Pietro 526201 1.9
LuizOtavio 341529 2.4
Murilo 273421 2.1
Oliver 506746 7.7
MariaSophia 946066 6.2
Alicia 549686 6.4
LuizFelipe 803204 6.8
Milena 346446 6.0
Isabel 618942 6.1
Benjamin 986373 8.5
Otavio 197447 4.1
Olivia 279368 8.2
Heitor 978095 6.5
ArthurGabriel 611452 3.6
Luana 868906 9.9
Nicole 622353 9.3
Breno 652764 5.9
EnzoGabriel 427990 0.7
Beatriz 269800 2.5
Isabella 600906 3.6
Rafael 540650 2.4
Gabrielly 566997 1.1
AnaBeatriz 914100 6.3
Vinicius 795281 6.5
Guilherme 422137 8.2
AnaLuiza 966349 6.4
Ayla 997397 2.8
Isaac 697385 9.1
DaviLucas 536864 4.1
PedroLucas 303010 3.2
AnaJulia 264102 4.4
Liz 836952 3.4
Julia 747148 3.8
Sophie 171090 8.9
Catarina 287649 9.5
MariaValentina 395657 0.7
Rafaela 720018 2.9
Pedro 266327 7.1
ArthurHenrique 709792 2.3
Martin 192083 1.0
Manuela 336541 3.3
