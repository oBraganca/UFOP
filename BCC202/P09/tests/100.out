JoaoGuilherme 403042 0.6
MariaValentina 395657 0.7
EnzoGabriel 427990 0.7
LuizMiguel 975866 0.7
Martin 192083 1.0
Gabrielly 566997 1.1
Bianca 106958 1.2
VictorHugo 3713 1.3
LucasGabriel 38609 1.3
Emanuel 382454 1.4
Gael 601185 1.5
Kauê 784637 1.6
Ryan 752634 1.8
Ian 278622 1.9
Pietro 526201 1.9
Joana 812356 1.9
Murilo 273421 2.1
MariaFernanda 761050 2.1
Gabriela 909372 2.1
Elisa 550011 2.3
ArthurHenrique 709792 2.3
LuizOtavio 341529 2.4
Rafael 540650 2.4
JoaoGabriel 783559 2.4
Beatriz 269800 2.5
Livia 410218 2.5
PedroMiguel 317302 2.8
Ayla 997397 2.8
Rafaela 720018 2.9
Marina 44918 3.0
PedroLucas 303010 3.2
Fernanda 17160 3.3
Larissa 122344 3.3
Manuela 336541 3.3
Agatha 942503 3.3
Liz 836952 3.4
Isabella 600906 3.6
ArthurGabriel 611452 3.6
Enzo 802024 3.6
Julia 747148 3.8
AnaLaura 73060 4.0
Otavio 197447 4.1
Joao 378948 4.1
DaviLucas 536864 4.1
Erick 755993 4.3
AnaJulia 264102 4.4
Amanda 153881 4.6
Bernardo 168265 4.8
Mariah 376893 4.8
Pietra 31845 4.9
Nicolas 607739 5.0
JoaoLucas 631990 5.0
Thomas 416182 5.2
LuizGustavo 358286 5.3
Isis 25792 5.6
MariaEduarda 423549 5.9
Breno 652764 5.9
Milena 346446 6.0
Isabel 618942 6.1
MariaSophia 946066 6.2
Bruno 39210 6.3
AnaBeatriz 914100 6.3
Alicia 549686 6.4
AnaLuiza 966349 6.4
Vinicius 795281 6.5
Heitor 978095 6.5
Lucca 747818 6.7
LuizFelipe 803204 6.8
JoaoMiguel 631594 7.0
Victor 752759 7.0
Pedro 266327 7.1
Henry 613407 7.5
Caua 979635 7.6
Oliver 506746 7.7
Samuel 924683 7.9
Benicio 617490 8.0
Ruan 778027 8.0
AnaLiz 310209 8.1
Luna 605714 8.1
Olivia 279368 8.2
Guilherme 422137 8.2
Thiago 862749 8.5
Benjamin 986373 8.5
Alice 233725 8.7
Isadora 537797 8.8
Lucas 827345 8.8
Sophie 171090 8.9
Isaac 697385 9.1
Joaquim 187555 9.3
Leonardo 289192 9.3
Allana 447784 9.3
Nicole 622353 9.3
Theo 202437 9.5
Catarina 287649 9.5
Maitê 782422 9.5
AnaSophia 15646 9.6
Lais 778936 9.6
Francisco 319887 9.8
Luana 868906 9.9
Arthur 775731 10.0
